docs/source/articles/extension-overview.html.md

```diff
@@ -23,7 +23,7 @@ Workarea provides various _designed_ points of extension that employ conventions
 - [Content block types](add-a-content-block-type.html)
 - [Discount types](create-a-custom-discount.html)
 - Inventory policies
-- [Payment gateways](integrate-a-payment-gateway.html)
+- [Payment gateways](implement-the-credit-card-tender-type.html)
```

Comment:
name doesn't match title.

Reply:
Yeah, I should rename the extension point to "tender type". Since I don't have a generic "tender types" doc to link to yet, I'll continue linking to the credit card doc, but make it clear it is an example of a tender type.